PWA/src/views/Timesheet.vue

```diff
@@ -19,7 +19,7 @@
               </v-card-title>
               <v-card-text class="text-center subtitle-1 mt-3">
                 {{ $t('views_Timesheet_continue_desc0') }}
-                <strong> #{{ formId }}</strong><br />
```

Comment:
Just want to verify, was this intentional?

Reply:
yes, and adjusted continue existing form text 
![image](https://user-images.githubusercontent.com/43203096/83977284-46957100-a8b4-11ea-957a-ecd63df47dda.png)